src/hb-ot-layout.cc

```diff
@@ -1941,6 +1941,19 @@ hb_ot_layout_substitute_lookup (OT::hb_ot_apply_context_t *c,
 }
 
 #ifndef HB_NO_BASE
+/**
+ * hb_ot_layout_get_baseline:
+ * @font: a font
+ * @baseline: a baseline tag
+ * @direction: text direction.
+ * @script_tag:  script tag.
+ * @language_tag: language tag.
+ * @coord: (out): baseline value, INT_MAX when simulation also failed.
+ *
+ * Fetches a baseline value from the face, simulate if is possible.
+ *
+ * Return value: true only if the baseline is found inside the font.
```

Comment:
Maybe we can return a tristate enum instead, enum { NONE, EXISTS, SIMULATED } or something.

Reply:
No need for tristate.  If needed, is fine to "return false and output simulated".  We do that in some other API.